### PR TITLE
fix: add agung2001

### LINKED ISSUE #1565: [PROFILE ADDITION] <Agung Sundoro>
### Profile screenshot
Screenshot 2022-12-17 at 07-37-59 agung2001 - Overview

### PR DESCRIPTION
**Pull request by agung2001**
resolve #1565 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/agung2001.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/4479106/208213842-00ef8a2c-d048-4406-9c66-7569392956c2.png",
+    "issueId": 1565,
+    "name": "Agung Sundoro",
+    "username": "agung2001"
+}
```
Comment (Panquesito7) [inline profiles/agung2001.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/agung2001.json (hunk 1 above)
Comment (agung2001) [reply profiles/agung2001.json]: hi, @Panquesito7 thx for reviewing my PR 😄🙏
I've just [committed a new update](https://github.com/EddieHubCommunity/awesome-github-profiles/pull/1566/commits/5ea9ece929c224068e7d6b8a7510bbe7d069e6a0), I hope it'll fix the issue.
Just let me know if there's more to it. cheers mate 😄🍻

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**